packages/mobile/src/app/App.tsx

```diff
@@ -52,32 +57,42 @@ BigNumber.config({
 // }
 
 export class App extends React.Component {
+  state: State = {
+    reactInitTime: undefined,
+    reactLoadTime: undefined,
+  }
+
+  appStartListener: EmitterSubscription = new NativeEventEmitter().addListener(
```

Comment:
Is there a chance that by the time React has created App and this listener is hooked on, we've missed the event? Should this go in index.js instead?

Reply:
Good question - the event is emitted from `onReactContextInitialized` in MainActivity so I think that means the order is ok. We tested using a physical device and emulator without issue.

Is there an edge case you have in mind for why it might work on some devices but not others?

Reply:
No particular edge case in mind, just thinking out loud.
If it works it works :)